src/generator.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+function generateNumber() {
+  let generatedNumber = '';
+
+  while (generatedNumber.length < 4) {
+    const randomDigit = (Math.random() * 10).toString().slice(0, 1);
```

Comment:
Your logic is correct, but not reusable and hard to update. I'd take something more general, [like getRandomIntInclusive here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random#getting_a_random_integer_between_two_values)